api/server/routecache/cacher.go

```diff
@@ -0,0 +1,101 @@
+package routecache
+
+import (
+	"context"
+	"errors"
+	"github.com/iron-io/functions/api/models"
+	"github.com/iron-io/runner/common"
+	"math"
+	"sync"
+)
+
+type Cacher interface {
+	PrimeCache(ctx context.Context)
```

Comment:
Right - I wrote assuming a context I haven't laid down.

IMHO, the issue here is PrimeCache... It is the role of the server to prime the cache, and not that the cache should prime itself.